### DISCUSSION TITLE
Submit form from one component to another

### DISCUSSION THREAD

**Discussion by harrisonratcliffe** [livewire/livewire#5148] [1. Help] [answered]
Hello, I have a livewire component where my modal is and there is a form. 

I'd like to transfer/share/submit the data from that form to another function in another component. I know this is possible using $emit() and wire:click but is this possible with wire:submit?

**Comment by harrisonratcliffe** [comment, marked as answer]
I sorted this by emiting to another event and passing $this:
```
$this->emit('myEvent', $this);
```

Then I accessed the submitted form data in the other component.

**Comment by joshhanley** [comment]
@harrisonratcliffe I'd strongly suggest not sending `$this` in the event, as that whole component and its data needs to get serialised and sent to the front end for this to work, including potentially properties and data that shouldn't be sent.

I would just pass the data that you need to the event or create an array of the data and pass that to the event.
